docs/core/entity/climate.md

```diff
@@ -245,6 +245,10 @@ class MyClimateEntity(ClimateEntity):
 
 ### Set temperature
 
+:::note
+`ClimateEntity` has built-in validation that the target low temperature is lower than equal to target high temperature so integrations does not need to validate it in their own implementation.
```

Comment:
What happened to the previous version and my suggestion? I suggest we bring my suggestion back. 🙂

Reply:
Yeah. My rebase erased that previous change by accident. I will change it again 👍